app/jobs/unseen_comments_notification_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class UnseenCommentsNotificationJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    User.includes(:comments).where(comments: { seen: false }).distinct.each do |user|
+      comments = user.comments.where(seen: false).to_a
+      CommentMailer.notification_user_of_new_comments(user, comments).deliver_later
+      Comment.where(user_id: user.id, seen: false).update(seen: true)
```

Comment:
why did you do this if you have the same comment here https://github.com/VovchikD/post/pull/19/files#diff-304b7bbacba2cda177ae1a52302821b0a4b1843555a2ff0a6624350740c894e7R8

Reply:
I don't have that code there. This code so that after sending email mark comments seen

Reply:
explained